wallets/shared/src/rango.ts

```diff
@@ -154,6 +154,8 @@ export const KEPLR_COMPATIBLE_WALLETS: string[] = [
   WalletTypes.LEAP_COSMOS,
 ];
 
+export const DEFAULT_COSMOS_RPC_URL = 'https://rpc.cosmos.directory/cosmoshub';
+
```

Comment:
Please modify the RPC URL and relocate it within the Math wallet package.

Reply:
I moved it here because it is not something specific to MathWallet and can be used for any Cosmos transaction. If you still think I should move it let me know.